Python-Home-Challenges/app.py

```diff
@@ -0,0 +1,10 @@
+f = open("Challenge_01.txt", "r")
```

Comment:
When opening file it is mandatory to close it when its no longer needed. you can use "close" method, but better implementation would be to use "with...open"

Reply:
used 'with.... open'